packages/apps/lightning-ui-docs/.storybook/preview.js

```diff
@@ -41,7 +41,11 @@ const preview = {
       sort: 'requiredFirst'
     },
     docs: {
-      theme: themes.dark
+      theme: themes.dark,
+      argTypes: {
+        sort: 'alpha',
+        exclude: ['mode']
```

Comment:
excludes 'mode' props from all tables, but the catch is if you have `exclude` in a story, that array will override this.